### PR TITLE
Adds "you" to local visible_messages

### PR DESCRIPTION
**Pull request by kingofkosmos**
## About The Pull Request
Replaces [target], [user], etc. to "you" wherever it is the player who is receiving the message locally, mostly in combat situations.

**Before:**
**`John Johnson hits Smith Smithson!`**
and
**`Smith Smithson knocks on the reinforced window.`**

**After:**
**`John Johnson hits you!`**
and
**`You knock on the reinforced window.`**

Also makes some of the attack verbs to present tense ("hits" instead of "has hit").

## Why It's Good For The Game
Makes quickly glancing the chat more clear about who's doing what to whom in various situations.

### CODE REVIEW

**Comment by Cyprex** [comment]
Does this make "You hits thing" or "You hit thing"

**Comment by kingofkosmos** [comment]
> Does this make "You hits thing" or "You hit thing"

I'm not totally sure what you mean, but I'm aiming for mostly "Someone hit you", because it also keeps open for more different phrasing for attack verbs like "Someone hits you" as well.

**Comment by ShadowDeath6** [comment]
I think they mean object damage like hitting a door or a light with a toolbox

**Comment by kingofkosmos** [comment]
Oh yeah, it does change that! So before:
**`John Johnson hits the reinforced window with the fire axe!`**

Would now be:
**`You hit the reinforced window with the fire axe!`**

I was surprised myself, didn't realize to test that. I'm guessing it's this here what affects the change:
https://github.com/tgstation/tgstation/pull/45631/files#diff-638d9f4bf631ba885e06844ac34d7196R87

**Comment by kingofkosmos** [comment]
Ready for reviewing.

**Comment by PKPenguin321** [comment]
Might make it harder to parse logs on the users end if they change names during the round

**Comment by actioninja** [comment]
I'm really not sure what you mean by that @PKPenguin321

File: code/datums/martial.dm [hunk 1]
```diff
@@ -53,25 +53,25 @@
...
+			"<span class='userdanger'>[A] [atk_verb]ed you!</span>", null, COMBAT_MESSAGE_RANGE)
 
 	D.apply_damage(damage, A.dna.species.attack_type, affecting, armor_block)
 
 	log_combat(A, D, "punched")
 
 	if((D.stat != DEAD) && damage >= A.dna.species.punchstunthreshold)
-		D.visible_message("<span class='danger'>[A] has knocked [D] down!!</span>", \
-								"<span class='userdanger'>[A] has knocked [D] down!</span>")
+		D.visible_message("<span class='danger'>[A] knocked [D] down!!</span>", \
```
Comment (actioninja) [inline code/datums/martial.dm]: knocked > knocks

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 2]
```diff
@@ -35,17 +35,20 @@
...
 	if(M == user)
-		user.visible_message("<span class='notice'>[user] swallows a gulp of [src].</span>", "<span class='notice'>You swallow a gulp of [src].</span>")
+		user.visible_message("<span class='notice'>[user] swallows a gulp of [src].</span>", \
+			"<span class='notice'>You swallow a gulp of [src].</span>")
 		if(HAS_TRAIT(M, TRAIT_VORACIOUS))
 			M.changeNext_move(CLICK_CD_MELEE * 0.5) //chug! chug! chug!
 
 	else
-		M.visible_message("<span class='danger'>[user] attempts to feed the contents of [src] to [M].</span>", "<span class='userdanger'>[user] attempts to feed the contents of [src] to [M].</span>")
+		M.visible_message("<span class='danger'>[user] is feeding [M] the contents of [src].</span>", \
```
Comment (actioninja) [inline code/modules/food_and_drinks/drinks/drinks.dm]: I don't like the phrasing on this. back to attempts to feed would make more sense for what's happening.

File: code/modules/food_and_drinks/drinks/drinks/bottle.dm [hunk 3]
```diff
@@ -100,11 +100,11 @@
 
 	//Display an attack message.
 	if(target != user)
-		target.visible_message("<span class='danger'>[user] has hit [target][head_attack_message] with a bottle of [src.name]!</span>", \
-				"<span class='userdanger'>[user] has hit [target][head_attack_message] with a bottle of [src.name]!</span>")
+		target.visible_message("<span class='danger'>[user] hits [target][head_attack_message] with a bottle of [src.name]!</span>", \
+				"<span class='userdanger'>[user] hits you[head_attack_message] with a bottle of [src.name]!</span>")
```
Comment (actioninja) [inline code/modules/food_and_drinks/drinks/drinks/bottle.dm]: missing a space

File: code/modules/food_and_drinks/food/condiment.dm [hunk 4]
```diff
@@ -46,14 +46,17 @@
 		return 0
 
 	if(M == user)
-		user.visible_message("<span class='notice'>[user] swallows some of the contents of \the [src].</span>", "<span class='notice'>You swallow some of the contents of \the [src].</span>")
+		user.visible_message("<span class='notice'>[user] swallows some of the contents of \the [src].</span>", \
+			"<span class='notice'>You swallow some of the contents of \the [src].</span>")
 	else
-		user.visible_message("<span class='warning'>[user] attempts to feed [M] from [src].</span>")
+		M.visible_message("<span class='warning'>[user] is feeding [M] from [src].</span>", \
```
Comment (actioninja) [inline code/modules/food_and_drinks/food/condiment.dm]: same as the other message

File: code/modules/food_and_drinks/food/snacks.dm [hunk 5]
```diff
@@ -118,18 +118,18 @@ All foods are distributed among various categories. Use common sense.
 		else
 			if(!isbrain(M))		//If you're feeding it to someone else.
 				if(fullness <= (600 * (1 + M.overeatduration / 1000)))
-					M.visible_message("<span class='danger'>[user] attempts to feed [M] [src].</span>", \
-										"<span class='userdanger'>[user] attempts to feed [M] [src].</span>")
+					M.visible_message("<span class='danger'>[user] is feeding [M] [src].</span>", \
+										"<span class='userdanger'>[user] is feeding you [src].</span>")
```
Comment (actioninja) [inline code/modules/food_and_drinks/food/snacks.dm]: ditto

**Review by actioninja — CHANGES_REQUESTED** [review]
mostly just grammar fixes

**Comment by PKPenguin321** [comment]
Last I checked you can save your own client side text logs from the BYOND client, and if a user has multiple different names in a round it may be harder for them to locate specific lines
On second thought that is such an esoteric problem that it probably won't ever happen

**Review by actioninja — APPROVED** [review]

**Pull request merged**